common/logging/include/claragenomics/logging/logging.hpp

```diff
@@ -69,7 +71,13 @@
 #endif
 
 // MUST come after the defines of the logging level!
+#if !(CGA_CUDA_BEFORE_9_2)
```

Comment:
I would stick with `#ifdef CGA_CUDA_BEFORE_`, i.e. use `#ifdef` and not invert the bool, such that the special handling happens right after the `#ifdef`.